tests/test_utils.py

```diff
@@ -1,9 +1,3 @@
-#!/usr/bin/env python
```

Comment:
I noticed this on other test files. We could probably remove it as well from all of them. But I’m fine with keeping that out of the scope of these changes.

Reply:
removed remaining shebangs

Reply:
Can you please also remove `if __name__ == '__main__':     unittest.main()` blocks from those test files?